packages/material-ui-lab/src/Autocomplete/Autocomplete.js

```diff
@@ -26,22 +26,12 @@ export const styles = theme => ({
   tag: {
     margin: 3,
     maxWidth: 'calc(100% - 6px)',
-    '&:last-child': {
-      marginRight: 5,
-      maxWidth: 'calc(100% - 8px)',
-    },
   },
   /* Styles applied to the tag elements, e.g. the chips if `size="small"`. */
   tagSizeSmall: {
     margin: 2,
     maxWidth: 'calc(100% - 4px)',
   },
-  /* Styles applied to the label of the tag elements, e.g. the chip's label. */
-  tagLabel: {
```

Comment:
@oliviertassinari Do you already have a PR for that? Or should I create create it?

Reply:
I believe we have a contributor already working on it as we are speaking :).

Reply:
Cool! Let me know if help is needed there. Because that change will make the chip look very broken if we don't ship the "label" part with it 😄

Reply:
Hopefully, it will be solved before the next release.

Reply:
It was :)